packages/overlay/test/overlay-trigger-hover.test.ts

```diff
@@ -53,32 +52,20 @@ async function styledFixture<T extends Element>(
 }
 
 describe('Overlay Trigger - Hover', () => {
-    afterEach(async () => {
-        const triggers = [
-            ...document.querySelectorAll('overlay-trigger'),
-        ] as OverlayTrigger[];
-        const promises = triggers.map((trigger) => {
-            if (trigger.open) {
-                const closed = oneEvent(trigger, 'sp-closed');
-                trigger.open = undefined;
-                return closed;
-            }
-            return Promise.resolve();
-        });
-        await Promise.all(promises);
-    });
-    beforeEach(async () => {
-        // Something about this prevents Chromium from swallowing the CSS transitions
-        // to "open" so that test timing can be properly acquired below.
-        await sendMouse({
-            steps: [
-                {
-                    type: 'move',
-                    position: [0, 0],
-                },
-            ],
-        });
-    });
+    // afterEach(async () => {
+    //     const triggers = [
+    //         ...document.querySelectorAll('overlay-trigger'),
+    //     ] as OverlayTrigger[];
+    //     const promises = triggers.map((trigger) => {
+    //         if (trigger.open) {
+    //             const closed = oneEvent(trigger, 'sp-closed');
+    //             trigger.open = undefined;
+    //             return closed;
+    //         }
+    //         return Promise.resolve();
+    //     });
+    //     await Promise.all(promises);
+    // });
```

Comment:
```suggestion
```

Reply:
Whoops, I got too excited when that actually fixed the issue I was running into.